gocourse03/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
```

Comment:
```suggestion
	"math/rand/v2"
```

Стаття https://go.dev/blog/randv2 пояснює, чому `math/random/v2` краще за `math/random`.